addon/ng2/blueprints/ng2/files/__path__/polyfills.ts

```diff
@@ -0,0 +1,4 @@
+// Prefer CoreJS over the polyfills above
+import 'core-js/es6';
```

Comment:
we probably don't need the entire core-js/es6 lib here - let's not polyfill the world unnecessarily?